forge/src/main/java/org/openjdk/skara/forge/github/GitHubRepository.java

```diff
@@ -582,7 +582,9 @@ public Optional<HostedCommit> commit(Hash hash, boolean includeDiffs) {
         } else {
             diffs = List.of();
         }
-        return Optional.of(new HostedCommit(metadata, diffs, URI.create(o.get("html_url").asString())));
+        var url = URI.create(o.get("html_url").asString());
+        var webUrl = gitHubHost.getWebURI(url.getPath());
+        return Optional.of(new HostedCommit(metadata, diffs, url, webUrl));
```

Comment:
Should we provide an implementation for GitLab as well?

Reply:
No I don't think so as we currently don't have a need to rewrite any GitLab URLs. We rewrite `github.com` to `git.openjdk.org` so that links stored in OpenJDK infrastructure (like JBS, mailing lists, etc.) aren't tried to GitHub (if we ever change forge then we can implement a re-direct service). For GitLab we currently don't have that need.